packages/server-core/src/social/invite/invite.resolvers.ts

```diff
@@ -93,7 +109,10 @@ export const inviteDataResolver = resolve<InviteType, HookContext>(
       return v4()
     },
     createdAt: getDateTimeSql,
-    updatedAt: getDateTimeSql
+    updatedAt: getDateTimeSql,
+    passcode: async () => {
```

Comment:
This should be above createdAt: to keep things clean

Reply:
implemented!